test/benchmarks/SunSpider/3d-cube.js

```diff
@@ -20,9 +20,16 @@
  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
  OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+if(typeof(WScript) === "undefined")
+{
+    var WScript = {
+        Echo: print
```

Comment:
@obastemur That looks better. Can we do that in a future change? This one is copied from other benchmark files. To make that change we better change them all.